easybuild/tools/utilities.py

```diff
@@ -99,5 +102,9 @@ def import_available_modules(namespace):
                 mod_name = module.split(os.path.sep)[-1].split('.')[0]
                 modpath = '.'.join([namespace, mod_name])
                 _log.debug("importing module %s" % modpath)
-                modules.append(__import__(modpath, globals(), locals(), ['']))
+                try:
+                    mod = __import__(modpath, globals(), locals(), [''])
+                except ImportError as err:
```

Comment:
why do you want to mask this ImportError?
Isn't it better to just rais the ImportError, so the function that uses this method can try to handle it?
Now it has to Catch an EasyBuildError, and it doesn't know what that means, anything could have gone wrong, since EasyBuildErrors are raised all over the place, so it can't try to recover from the error but is forced to quit?